src/platform/silabs/wifi/wiseconnect-abstraction/WiseconnectInterfaceAbstraction.cpp

```diff
@@ -50,6 +50,37 @@ CHIP_ERROR GetMacAddress(sl_wfx_interface_t interface, chip::MutableByteSpan & a
     return CopySpanToMutableSpan(byteSpan, address);
 }
 
+CHIP_ERROR StartNetworkScan(chip::ByteSpan ssid, ScanCallback callback)
```

Comment:
Do you have a plan to unify these almost identical implementations of the this function?